### PR TITLE
Homework6

### PR DESCRIPTION
**Pull request by OlegShvets1**
Homework 6

### CODE REVIEW

File: src/Lesson_6/Task_1/Tv_1.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package Lesson_6.Task_1;
+
+public class Tv_1 {
+    public static String channel;
```
Comment (Evangard) [inline src/Lesson_6/Task_1/Tv_1.java:4]: No need to make it static.

File: src/Lesson_6/Task_1/Radio_1.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package Lesson_6.Task_1;
+
+public class Radio_1 {
+    public static String music;
+    public static String news;
```
Comment (Evangard) [inline src/Lesson_6/Task_1/Radio_1.java:5]: No need to make it static.

File: src/Lesson_6/Task_3/Car.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+    private int engine;
+    private int weight;
+
+    public Car(String name, String size, int engine, int weight) {
+        super(name, size);
+        this.engine = engine;
+        this.weight = weight;
+    }
+    Vehicle vehicle;
```
Comment (Evangard) [inline src/Lesson_6/Task_3/Car.java:13]: You don't need it here.

File: src/Lesson_6/Task_3/Hyundai.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package Lesson_6.Task_3;
+
+public class Hyundai extends Car {
+    private String fuelType;
+
+    public Hyundai(String fuelType) {
+        super("Hyundai Santa Fe", "2114*5444", 3000, 2400);
```
Comment (Evangard) [inline src/Lesson_6/Task_3/Hyundai.java:7]: Don't use hardcode. Please, move it to the parameters of your constructor. E.g. public Hyundai(String name, String size, String engine, String weight, String fuelType)

File: src/Lesson_6/Task_4/Rose.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        this.highth = highth;
+    }
+
+    @Override
+    public void bloom() {
+        System.out.println("A rose is a flower, and therefore it also blooms");
+    }
+
+    @Override
+
```
Comment (Evangard) [inline src/Lesson_6/Task_4/Rose.java:24]: Remove this empty line.

File: src/Lesson_6/Task_5/Circle.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
+package Lesson_6.Task_5;
+
+public class Circle extends Figure {
+
+    public Circle(double width) {
+        this.width = width;
+
+    }
+
+    public  void calculateArea() {
```
Comment (Evangard) [inline src/Lesson_6/Task_5/Circle.java:10]: Don't forget about @Override annotation.

File: src/Lesson_6/Task_5/Rectangle.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+public class Rectangle extends Figure {
+
+    public Rectangle(double width, double heigth) {
+        this.width = width;
+        this.height = heigth;
+    }
+
+    public  void calculateArea() {
+        double S;
```
Comment (Evangard) [inline src/Lesson_6/Task_5/Rectangle.java:11]: Naming convention.

File: src/Lesson_6/Task_5/Triangle.java [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+public class Triangle extends Figure {
+
+    public Triangle(double width, double heigth) {
+        this.width = width;
+        this.height = heigth;
+    }
+
+    public void calculateArea() {
+        double S;
```
Comment (Evangard) [inline src/Lesson_6/Task_5/Triangle.java:11]: Naming convention.

File: src/Lesson_6/Task_1/Tv_1.java (hunk 1 above)
Comment (OlegShvets1) [reply src/Lesson_6/Task_1/Tv_1.java:4]: +

File: src/Lesson_6/Task_1/Radio_1.java (hunk 2 above)
Comment (OlegShvets1) [reply src/Lesson_6/Task_1/Radio_1.java:5]: +

File: src/Lesson_6/Task_3/Car.java (hunk 3 above)
Comment (OlegShvets1) [reply src/Lesson_6/Task_3/Car.java:13]: +

File: src/Lesson_6/Task_3/Hyundai.java (hunk 4 above)
Comment (OlegShvets1) [reply src/Lesson_6/Task_3/Hyundai.java:7]: +

File: src/Lesson_6/Task_4/Rose.java (hunk 5 above)
Comment (OlegShvets1) [reply src/Lesson_6/Task_4/Rose.java:24]: +

File: src/Lesson_6/Task_5/Circle.java (hunk 6 above)
Comment (OlegShvets1) [reply src/Lesson_6/Task_5/Circle.java:10]: +

File: src/Lesson_6/Task_5/Rectangle.java (hunk 7 above)
Comment (OlegShvets1) [reply src/Lesson_6/Task_5/Rectangle.java:11]: +

Comment (Evangard) [reply src/Lesson_6/Task_5/Rectangle.java:11]: There is still S variable. Should be s.

File: src/Lesson_6/Task_5/Triangle.java (hunk 8 above)
Comment (Evangard) [reply src/Lesson_6/Task_5/Triangle.java:11]: There is still S variable. Should be s.

**Pull request merged**